let mongoose;_11d‍.x([["default",()=>_11d‍.o]]);_11d‍.w("mongoose",[["default",["mongoose"],function(v){mongoose=v}]]);

const connectDb = async () => {
  try {
    await mongoose.connect(
      "mongodb+srv://vaibhavojha34:8ogjvz4RO48KHp91@cluster0.vxzczqs.mongodb.net/BlogSpot"
    );
    console.log("We are connected to our database");
  } catch (error) {
    console.log("The error  has been occurred during connecting to database");
  }
};

_11d‍.d(connectDb);
